### PR TITLE
Website tweaks and fixes

### PR DESCRIPTION
**Pull request by stjanilofts**
### Multiple somewhat minor fixes/changes:

1. Sidebar updates position on screen resize on the about page.
2. Spacing for items inside AccordionItem had become a bit messed up. Should be good now.
3. Hide material links section when there are no links.
4. Fix color scheme on buttons in about page header.
5. Fix next.js links when used in Breadcrumbs.
6. Fix spacing issue on mobile for rich text content on some pages

## Why

1. Sidebar was stuck in initial place on screen resize.
2. The faq list/category cards were looking a bit off (in regards to spacing).
3. Related material links section was showing up in the mobile drawer even though there were no links.
4. After the header menu buttons were swapped for the new button component the color scheme changes stopped working.
5. The Breadcrumbs component was breaking next.js links.
6. It looked ok in desktop but mobile should have other spacing values.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against master before asking for a review

### CODE REVIEW

File: libs/island-ui/core/src/lib/Accordion/AccordionItem/AccordionItem.tsx [hunk 1]
```diff
@@ -38,6 +38,7 @@ export type AccordionItemBaseProps = {
   onClick?: () => void
   onBlur?: () => void
   onFocus?: () => void
+  boxProps?: BoxProps
```
Comment (atligudl) [inline libs/island-ui/core/src/lib/Accordion/AccordionItem/AccordionItem.tsx:41]: I don't think Accordion Items should be given the full API of the Box component

Comment (stjanilofts) [reply libs/island-ui/core/src/lib/Accordion/AccordionItem/AccordionItem.tsx:41]: I agree. I will change this.

File: libs/island-ui/core/src/lib/Text/Text.tsx [hunk 2]
```diff
@@ -85,7 +85,13 @@ export const Text = ({
       {React.Children.map<React.ReactNode, React.ReactNode>(
         children,
         (child: any) => {
-          if (typeof linkRenderer === 'function' && child?.props?.href) {
+          if (child?.type?.href && typeof linkRenderer === 'function') {
```
Comment (atligudl) [inline libs/island-ui/core/src/lib/Text/Text.tsx:88]: @DavidBachmann might want to verify this

**Review by atligudl — APPROVED** [review]
LGTM, tagged Davíð in one spot

File: libs/island-ui/core/src/lib/Text/Text.tsx (hunk 2 above)
Comment (DavidBachmann) [reply libs/island-ui/core/src/lib/Text/Text.tsx:88]: This is a weird one. IIRC `child.type` is undefined outside Next (?) so I opted for the mutual `props.href` instead. Try running Storybook and see if it works.

Comment (stjanilofts) [reply libs/island-ui/core/src/lib/Text/Text.tsx]: It should have been `child.href` not `child.type`. I have made some changes to this.

File: apps/web/screens/Article.tsx [hunk 3]
```diff
@@ -481,7 +485,7 @@ const ArticleScreen: Screen<ArticleProps> = ({ article, namespace }) => {
         <Box paddingTop={subArticle ? 2 : 4}>
           <RichText
             body={(subArticle ?? article).body as SliceType[]}
-            config={{ defaultPadding: 4 }}
+            config={{ defaultPadding: [2, 2, 4] }}
```
Comment (siggakogk) [inline apps/web/screens/Article.tsx:488]: haha 💯

**Review by DavidBachmann — APPROVED** [review]
LGTM

**Pull request merged**